src/audiobuffer-manager.js

```diff
@@ -19,6 +19,8 @@
 
 'use strict';
 
+var Utils = require('./utils');
```

Comment:
Can we add '.js' at the end of file path? I know it's not required, but I want to keep them consistent.